### ISSUE TITLE
Add Volley build-time autoinstrumentation

### ISSUE THREAD

**Issue by breedx-splk** [open-telemetry/opentelemetry-android#292]
After #291 is merged and the Volley library instrumentation exists, we can then look into adding build-time autoinstrumentation much like we have for okhttp.

**Comment by marandaneto** [comment]
I think it's not needed for v1 since this can be added later, it is not a behavior or API surface breakage, unless doing this requires changes in the API surface

**Comment by breedx-splk** [comment]
We want to deprecate Volley instrumentation, so we will just close this.
